lib/delta.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var util = require('util');
+var deltaPrefix = '\u2206';
+var altDeltaPrefix = '\u0394';
+
+module.exports = {
+  deltaCounterName : function(metricName) {
+    if (module.exports.hasDeltaPrefix(metricName)) {
+      return metricName;
+    }
+    return deltaPrefix + metricName;
+  },
+
+  hasDeltaPrefix : function(metricName) {
+    if (metricName) {
+      return metricName.startsWith(deltaPrefix);
+    }
+  },
```

Comment:
better to return false in the else case
`return metricName ? metricName.startsWith(deltaPrefix) : false`